src/psyclone/psyir/nodes/scoping_node.py

```diff
@@ -183,7 +183,9 @@ def reference_accesses(self) -> VariablesAccessMap:
         # During the updating process when moving a Routine (and its
         # associated Symbol), it's possible that we won't have a SymbolTable.
         if self._symbol_table:
-            var_accesses.update(self._symbol_table.reference_accesses())
+            for symbol in self._symbol_table.get_all_accessed_symbols():
```

Comment:
What about symbols in initialisation expressions? Strictly they are not TYPE_INFO accesses.